netci.groovy

```diff
@@ -73,27 +73,28 @@ class WcfUtilities
             return 
         }
 
-        wcfRepoSyncServiceCount++
+        wcfPRServiceCount++
```

Comment:
I kind of have the question for a while but never asked so I just ask explicitly so I don't worry anymore :). With this approach, do we guarantee each CI job is always assigned the same id?